src/components/radioButton/demoBasicUsage/script.js

```diff
@@ -31,8 +31,14 @@ angular
     ];
 
 
-    $scope.submit = function() {
-      alert('submit');
+    $scope.submit = function($event) {
+      $mdDialog.show(
+        $mdDialog
+          .alert()
+          .title('Action')
+          .textContent('submit')
+          .event($event)
```

Comment:
This throws
```
TypeError: $mdDialog.alert(...).title(...).textContent(...).event is not a function
    at m.$scope.submit (docs-demo-scripts.js:2746)
```

Reply:
I'm probably not understanding how $mdDialog works here, would be good for an angular material dev to take a look, please?

Reply:
It's .`targetEvent($event)`.